install-php-extensions

```diff
@@ -1511,7 +1515,14 @@ installMicrosoftSqlServerODBC() {
 				DEBIAN_FRONTEND=noninteractive apt-get -q update
 			fi
 			printf -- '- installing the APT package\n'
-			DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get -qy --no-install-recommends install '^msodbcsql[0-9]+$'
+			if test $PHP_MAJMIN_VERSION -le 703; then
+				DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get -qy --no-install-recommends install msodbcsql17
+			elif test $DISTRO_VERSION_NUMBER -ge 9 && $DISTRO_VERSION_NUMBER -le 10; then
+				# On Debian 9 and 10 we have both msodbcsql17 and msodbcsql18: let's install just one
+				DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get -qy --no-install-recommends install msodbcsql18
+			else
+				DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y apt-get -qy --no-install-recommends install '^msodbcsql[0-9]+$'
```

Comment:
This else still installs both versions on Debian 18.

https://github.com/mvorisek/image-php/commit/6bdcd5df242ba999685322f5e196db40dfef55b1

there should be probably no regex and only `msodbcsql18` until new major release of Debian and MSSQL ODBC is out

Reply:
Debian 18?

Reply:
Debian 11.2, a typo